lit_llama/lora.py

```diff
@@ -24,6 +60,24 @@ def __init__(
         lora_dropout: float,
         merge_weights: bool,
     ):
+        """Store LoRA specific attributes in a class.
+
+        Parameters
```

Comment:
My bad, in my projects I use numpy style.
Perhaps in one of the next PRs it would be great to add linters/auto-formatters to make codebase more cohesive.

Reply:
Haven't worked with google style before but I think I successfully converted docstrings.
The only thing that I am not fully sure is for multiple options (like for bias):
```text
Args:
    model: model with LoRA layers
    bias: 
        ``"none"``: state dict will not store bias weights,
        ``"lora_only"``: state dict will store bias weights only from LoRA layers,
        ``"all"``: state dict will store all bias weights.
```

Basically I replicated docstring from `generate/adapter.py`:

https://github.com/Lightning-AI/lit-llama/blob/5df20dbb2084f0406cf4a8f6a5de69b5017fcf89/generate/adapter.py#L43-L45